script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -351,7 +353,8 @@ The scream wrung from the victim of her blow seems to have driven her wild with
 -- Page 445, Offset 19376.
 -- ああもうグズが、モタモタしてんじゃねえよ！　あんな女、外にさえ出ればカンタンに”
 // VA is SCREECHING this one, give that extra a raise
-%{i}Ugh%{/i}, get a fucking move on, you slowpoke!
+// Play around with how to emphasize these nicely
```

Comment:
put my name or something in this if you would so i can grep for the line

Reply:
// TODO(ben) added